First version of casefile for DIVD-2025-0004 (#915)
DIVD-NL/web-csirt: _cases/2025/DIVD-2025-00004.md

---
layout: case
# Title and excerpt will be used on /cases and the RSS feed so make sure they reflect the case well
title: Authentication Bypass in PAN-OS Management Web Interface

> excerpt: "Please set this to and except of a few lines.

Reviewer: Misschien deze nog even?

> excerpt: "Please set this to and except of a few lines.

Reviewer 2: Missing excerpt

Author: Fixed

Excerpts are shown on /cases"
author: Frank Breedijk
lead: Frank Breedijk
researchers:
- Frank Breedijk
- Stijn Engelhard (intern)
# You can use free text here as well. E.g. to indicate that some vulnerabilities don't have CVEs assigned (yet).
cves:
- CVE-2025-0108
product: 
- Palo Alto PAN-OS
versions: 
- v11.2 (<11.2.4-h4)
- v11.1 (<11.1.2-h18 or <11.1.6-h1)
- v10.2 (<10.2.7-h24 or < 10.2.8-h21 or < 10.2.9-h21 or < 10.2.10-h14 or < 10.2.11-h12 or < 10.2.12-h6 or < 10.2.13-h3)
- 10.1 (< 10.1.14-h9)
recommendation: Install the updates provided by the vendor. Make sure your management interface is not exposed to the internet.
patch_status: Fully patched
#workaround: n/a
status : Open
start: 2025-02-20
end: 
timeline:
- start: 2025-02-12
  end:

>   event: "Palo Alto published adviosry PAN-273971 and released patches"

Reviewer: ```suggestion
  event: "Palo Alto published advisory PAN-273971 and released patches"
```

- start: 2025-02-12
  end:
  event: "Assetnote releases full vulnerability details"
- start: 2025-02-13
  end:
  event: "First attacks observed by GreyNoise"
- start: 2025-02-13
  end:
  event: "Nuclei template published on GitHub"
- start: 2025-02-14
  end:
  event: "DIVD Starts investigation"
- start: 2025-02-20
  end:
  event: "Case opened"
# ips: 0 
# ips is used for statistics after the case is closed. If it is not applicable, you can set IPs to n/a (e.g. stolen credentials)
# This field becomes mandatory when the case status is set to 'Closed'
# creds: 0 
# creds is used for statistics after the case is closed if the case involves stolen credentials. If it is not applicable, you can
# set it to n/a or 0 or leave this field unset.
---
## Summary

Due to confusion between the ngnix and apache web servers/proxies used to serve the PAN-OS web managmenet interface, it is possible to access certain PHP scripts on Palo Alto devices running PAN-OS without authentication as described in {% cve: CVE-2025-0108 %}. Running these scripts can lead to a compromise of the confidentiality and integrity of the device.

## What you can do

> It is high recommended not to publicly expose management interfaces of edge devices to the internet. THis is also the case for PAN-OS device as per their [best paractices deployment guidelines](https://live.paloaltonetworks.com/t5/community-blogs/tips-amp-tricks-how-to-secure-the-management-access-of-your-palo/ba-p/464431).

Reviewer: ```suggestion
It is highly recommended not to publicly expose management interfaces of edge devices to the internet. This is also the case for PAN-OS device as per their [best practices deployment guidelines](https://live.paloaltonetworks.com/t5/community-blogs/tips-amp-tricks-how-to-secure-the-management-access-of-your-palo/ba-p/464431).
```

> * If you have the managmenet interface of you PAN-OS device exposed to internet, please restrict access to trusted IP addresses only.

Reviewer: ```suggestion
* If you have the managmenet interface of your PAN-OS device exposed to the internet, please restrict access to trusted IP addresses only.
```

* Install vendor supplied patches

## What we are doing

> DIVD is currently scanning the internet for vulnerable devices. Once devices are identitied we will warn the administrators of the networks they are located in.

Reviewer: ```suggestion
DIVD is currently scanning the internet for vulnerable devices. Once devices are identified we will warn the administrators of the networks they are located in.
```
